Gopkg.toml

```diff
@@ -1,3 +1,4 @@
 [[constraint]]
     name = "github.com/pilosa/go-pilosa"
-    branch = "master"
\ No newline at end of file
+    branch = "imagine"
+    source = "github.com/jaffee/go-pilosa"
```

Comment:
gotta figure out why I was doing this and fix it... can probably use pilosa@master now